### DISCUSSION TITLE
secure safe in vault

### DISCUSSION THREAD

**Discussion by deltanedas** [space-wizards/space-station-14#20597] [Ideaguys]
from #20569

safe that can store high value things that must stay in the vault, and stuff like gold and silver reserves.

using a missile silo turnkey system like early launch console
2 heads have to authorize opening the safe within a short period of time say 10 seconds
as a syndie trying to break in you can:
- kill 2 heads and steal their ids
- trick them into doing it somehow
- hide in the fridge waiting for them to open it then jump out and fishbomb when they unlock
- emag to do something unique

for emagging:
- either emag twice to disable both authorization requirements which is pretty boring
- or what i think is a bit cooler, emagging to disable access check but still require 2 unique ids. so you can kill the chef and use his id to "authorize" opening the safe.

authorizations could be logged on a computer somewhere then looked at later (like #17810, maybe even using it)

if a syndie uses their id they would be very sus if its checked in logs afterwards which would incentivise getting multiple ids from dead/ssd people to use them as cover

it should also be significantly stronger than most structures, a single c4 shouldnt crack it open but a few might

**Comment by GoodWheatley** [comment]
I think the turnkey system is kind of pointless, if a syndie already did the work of getting into the vault they should be able to grab the stuff inside it without any more hoops to jump through.

**Reply by deltanedas** [reply]
the thing is, there is basically no work required to get into the vault:
1. you can just disposal into it easy peasy
2. you can turn an apc off then pry the door open, if its outside
3. you can just hack it open

**Reply by GoodWheatley** [reply]
You could just put a regular safe in the vault then, instead of this weird system that just makes it harder for syndies to do objectives.

**Reply by deltanedas** [reply]
if gold and silver are put in it too then its cooler having another head open it with rd when salv doesnt mine anything

**Reply by drteaspoon420** [reply]
I like the idea but it would result in power gaming.
